### ISSUE TITLE
Full hierarchy needed for cutat

### ISSUE THREAD

**Issue by gaborcsardi** [igraph/rigraph#58]
_From @Yquetzal on October 21, 2014 1:53_

The cutat method check for a fully hierarchical community decomposition. However, the leading.eigenvector method do not return a FULL hierarchy, so it does not work with it. Is it a bogue ? If no, is there another solution to get the communities at a given level for leading.eigenvector.community ?

_Copied from original issue: igraph/igraph#748_

**Comment by gaborcsardi** [comment]
Yes, it is not really a bug, I simply forgot about it. Will be done in the next release. There were workarounds on the igraph-help mailing list, AFAIR.

**Comment by zachmayer** [comment]
Similarly multilevel.community does not return a full hierarchy, but would still be useful to visualize as a dengrogram.
